doc/chapters/api/index.rst

```diff
@@ -80,7 +82,7 @@ Launches the specified application/code in the container.
 |
 
 :Return Values:
-        :pid: ``int32`` PID of the process run inside the container.
+        :pid: ``int32`` the external PID of the process run inside the container.
```

Comment:
Maybe "PID of process running in the container, as seen by the host" or something like that? "External" might be confusing for some readers I think.